projects/go-lib/src/lib/components/go-table/go-table.component.html

```diff
@@ -38,14 +38,14 @@ <h3 class="go-heading-3 go-heading--no-margin go-table-header__title" *ngIf="tab
 
           <!-- Header - Detail Row / Child Row Arrow Column -->
           <th
-            class="go-table__body-column"
+          class="go-table__body-column"
             *ngIf="details || childRows"
             width="40">
           </th>
 
           <!-- Header - Select All Checkbox Column -->
           <th
-            class="go-table__head-column go-table__head-checkbox go-table__head--sortable"
+            class="go-table__head-column go-table__head-checkbox go-table__head--sortable {{classStickyHeader}}"
```

Comment:
There really should never be a reason to interpolate things into the `class` attribute. For this you'll want to use `ngClass`.  See [the docs](https://angular.io/api/common/NgClass).